packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -739,9 +739,10 @@ export class StripeHelper extends StripeHelperBase {
     automaticTax: boolean;
     subscriptionId: string;
   }) {
+    let invoice: Stripe.Invoice;
     if (automaticTax) {
       try {
-        return this.stripe.invoices.retrieveUpcoming({
+        invoice = await this.stripe.invoices.retrieveUpcoming({
```

Comment:
I would recommend `return await` to utilize your try/catch rather than a shared variable.

Reply:
done